### PR TITLE
bugfix/CLDSRV-264/ignore_lifecycle_event_in_utapi

### PR DESCRIPTION
**Pull request by watsaqat**
# Pull request template

### Motivation and context

Addresses https://scality.atlassian.net/browse/S3C-5994. Drops lifecycle events so they are not pushed to utapi when the computed event objectDelta and sizeDelta are both 0.

### Add tests to cover the changes

New tests added or existing tests modified to cover all changes

### Sign your work

In order to contribute to the project, you must sign your work
https://github.com/scality/Guidelines/blob/master/CONTRIBUTING.md#sign-your-work

Thank you again for contributing! We will try to test and integrate the change
as soon as we can.

### CODE REVIEW

File: tests/utapi/utilities.js [hunk 1]
```diff
@@ -507,4 +513,51 @@ describe('utapi v2 pushmetrics utility', () => {
...
+            return event;
+        })
+        .forEach(event => {
+            if (!eventFilterList.has(event.action)) {
+                it(`should compute and push metrics for ${event.action}`, () => {
+                    const eventPushed = pushMetric(event.action, log, event.metrics);
+                    assert(eventPushed);
+                    Object.keys(event.expected).forEach(key => {
+                        assert.strictEqual(eventPushed[key], event.expected[key]);
+                    });
```
Comment (miniscruff) [inline tests/utapi/utilities.js:558]: Should this be `assert.strictDeepEquals`? Or is some stuff in event pushed optional? Maybe?

**Review by miniscruff — APPROVED** [review]

**Review by nicolas2bert — APPROVED** [review]

**Pull request merged**